components/Quote/Quote.module.css

```diff
@@ -0,0 +1,28 @@
+.largecontainer,
+.container {
+	margin: 1em 0;
+	padding: 1em;
```

Comment:
Some weird spacing here. Need to turn on linting via GitHub actions (and npm scripts).

Reply:
(Can be done separately)

Reply:
I **think** I fixed it around the same time as your comment

Reply:
VS Code won the battle of the tabs :-( it's all spaces now